### PR TITLE
Inital dev

### PR DESCRIPTION
**Pull request by LoicLEMEUT**

### CODE REVIEW

File: templates/generateHomeLogin.html.twig [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                <div class="panel-body">
+                    {{ sendinblue_send_email_form }}
+                </div>
+            </div>
+        </div>
+
+
+    </div>
+
+    {% include 'modules/contrib/sendinblue/templates/footer.html.twig' %}
```
Comment (amarpanday) [inline templates/generateHomeLogin.html.twig]: @LoicLEMEUT : Please remove '/contrib' from the path

File: templates/generateHomeLogout.html.twig [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+    </div>
+
+    {% include 'modules/contrib/sendinblue/templates/footer.html.twig' %}
```
Comment (amarpanday) [inline templates/generateHomeLogout.html.twig]: @LoicLEMEUT : Please remove '/contrib' from the path

**Review by amarpanday — COMMENTED** [review]
Please make the changes as per comment

File: templates/generateHomeLogin.html.twig (hunk 1 above)
Comment (LoicLEMEUT) [reply templates/generateHomeLogin.html.twig]: @amarpanday Done

**Comment by LoicLEMEUT** [comment]
@amarpanday all Done

**Review by amarpanday — APPROVED** [review]
@LoicLEMEUT  LGTM :+1:

**Pull request merged**